pvm/ts/src/engine/actions/raiseAction.test.ts

```
@@ -116,23 +112,43 @@ describe("Raise Action", () => {
             seat: 2,
             timestamp: Date.now()
         };
 
         beforeEach(() => {
-            const now = Date.now();
             jest.spyOn(game, "getPlayersLastAction").mockReturnValue(lastBet);
             jest.spyOn(game, "getPlayerSeatNumber").mockReturnValue(3);
         });
 
-        it("should return correct range for a raise", () => {
+        it.only("should return correct range for a raise", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)